docs/en/development-guide.md

```diff
@@ -0,0 +1,82 @@
+# Development guide
+
+## Run your first build
+
+Follow [the "Run from source code" section](../../README.md) on README.
```

Comment:
Maybe we can use `../../README.md#run-from-source-code`?

Reply:
Fixed